### PR TITLE
[containerapp] `az containerapp env create`: Update the default value of `--enable-workload-profiles` to `True`

### PR DESCRIPTION
**Pull request by Greedygre**
**Related command**

az containerapp env create: --enable-workload-profiles default value change to True
az containerapp compose create: fix containerapp invalid memory resource
Change the value of polling Timeout from 10min to 20min.

**Description**

**Testing Guide**

**History Notes**

[containerapp] BREAKING CHANGE: `az containerapp env create`: Update the default value of `--enable-workload-profiles` to `True`
[containerapp] `az containerapp compose create`: Fix containerapp invalid memory resource

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py [hunk 1]
```diff
@@ -358,6 +360,39 @@ def test_containerapp_env_workload_profiles_e2e_no_waits(self, resource_group):
         self.cmd('containerapp delete -g {} -n {} --yes'.format(resource_group, app2), expect_failure=False)
         self.cmd('containerapp env delete -g {} -n {} --yes'.format(resource_group, env), expect_failure=False)
 
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="eastus")
+    def test_containerapp_env_enable_workload_profiles_infer_env_type(self, resource_group):
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+        env = self.create_random_name(prefix='env', length=24)
+        self.cmd('containerapp env create -g {} -n {} --logs-destination none --enable-workload-profiles false'.format(
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py:368]: create on existing v2 cluster

**Comment by Greedygre** [comment]
Hi @yonzhan @wangzelin007 

The CI pipeline failed due to ofollowing error, could you help to handle it? Thanks!

```
DEBUG: cli.knack.cli: Event: MainLoader.OnFailedExtensionLoad []
WARNING: cli.azure.cli.core: Unable to load extension 'alias: cannot import name 'soft_unicode' from 'markupsafe' (/opt/az/azcliextensions/alias/markupsafe/__init__.py)'. Use --debug for more information.
DEBUG: cli.azure.cli.core: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.6/x64/lib/python3.11/site-packages/azure/cli/core/__init__.py", line 351, in _update_command_table_from_extensions
    _load_extension_command_loader(self, args, ext_mod)
  File "/opt/hostedtoolcache/Python/3.11.6/x64/lib/python3.11/site-packages/azure/cli/core/commands/__init__.py", line 1081, in _load_extension_command_loader
    return _load_command_loader(loader, args, ext, '')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.6/x64/lib/python3.11/site-packages/azure/cli/core/commands/__init__.py", line 1052, in _load_command_loader
    module = import_module(prefix + name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.6/x64/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/az/azcliextensions/alias/azext_alias/__init__.py", line 15, in <module>
    from azext_alias._validators import (
  File "/opt/az/azcliextensions/alias/azext_alias/_validators.py", line 13, in <module>
    from azext_alias.argument import get_placeholders
  File "/opt/az/azcliextensions/alias/azext_alias/argument.py", line 13, in <module>
    import jinja2 as jinja
  File "/opt/az/azcliextensions/alias/jinja2/__init__.py", line 12, in <module>
    from .environment import Environment
  File "/opt/az/azcliextensions/alias/jinja2/environment.py", line 25, in <module>
    from .defaults import BLOCK_END_STRING
  File "/opt/az/azcliextensions/alias/jinja2/defaults.py", line 3, in <module>
    from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/az/azcliextensions/alias/jinja2/filters.py", line 13, in <module>
    from markupsafe import soft_unicode
ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/opt/az/azcliextensions/alias/markupsafe/__init__.py
```

**Comment by Greedygre** [comment]
Hi @yanzhudd 

Could you please help to review and merge this PR? Thanks!!

**Comment by yanzhudd** [comment]
Since this is a breaking change, please add "BREAKING CHANGE" to the history notes.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**